src/validators/dataclass.rs

```diff
@@ -453,6 +454,8 @@ impl BuildValidator for DataclassValidator {
             )?)?,
             name,
             frozen: schema.get_as(intern!(py, "frozen"))?.unwrap_or(false),
+            slots: matches!(class.hasattr(intern!(class.py(), "__slots__")), Ok(true))
+                | schema.get_as(intern!(py, "slots"))?.unwrap_or(false),
```

Comment:
So, Then we have to specify `slots=True` in the schema always and it can't be taken from 
`@dataclasses.dataclass(slots=True)`

Is it the desire behavior?

Reply:
yes. I'll implement now.